DEPLOYMENT.md

```diff
@@ -35,52 +46,64 @@ One for the Java server, one for hosting the client files, and one for the datab
-- Scroll down and choose "One-time password." This will have them email you a temporary password for the `root` user.
+- Scroll down and choose "Password" under Authentication. Enter a password here,
+  this will be the password for the `root` user.
+  - This should be a good, secure password since it gives access to everything on
+    your droplet and anyone can attempt to `ssh` into it. You may wish to use a random
+    password generator for this.
+  - You can change this password later with the `passwd` command.
 - You don't need to add block storage or backups.
 - Finally, only make one droplet and choose a name for it.
 - It will take a couple seconds to make the droplet.
-- You will then be able to see it on your [Droplets](https://cloud.digitalocean.com/droplets) page and get the IP for it.
-
+- You will then be able to see it on your
+  [Droplets](https://cloud.digitalocean.com/droplets) page and get the IP for it.
 
 ## Step 3: Setting up your droplet and running your project
-- SSH to your droplet by running ``ssh root@[my ip here]`` (using the IP of your droplet) and enter the password emailed to you.
-- It will ask you to set a new password for `root`. Enter the original password emailed to you and then set a new one. 
-  - This should be a good, secure password since it gives access to everything on your droplet and anyone can attempt to `ssh` into it. You may wish to use a random password generator for this.
-  - You can change this password later with the `passwd` command.
+
+- SSH to your droplet by running ``ssh root@[my ip here]`` (using the IP of your droplet) and enter the password you set.
+- When you first log in it'll tell you if there are any updates available. If there are, you can run `apt update` and then `apt upgrade` to apply the updates.
 - `git clone` your repository
 - `cd` into the newly created directory
 - run `./setupdroplet.sh` to go through the initial setup steps
-  - It will ask you to agree to the [Let's Encrypt Subscriber Agreement](https://letsencrypt.org/documents/2017.11.15-LE-SA-v1.2.pdf). You will need to answer `yes` or `y` to have HTTPS enabled.
-  - If you answered yes in the last step it will ask for an email address, this is needed for HTTPS as well and will be used for any relevant alerts about your HTTPS certificate (you probably won't get any emails from them).
-  - We are using a service called [nip.io](https://nip.io/) to give us the valid domains we need for HTTPS. The script will tell you the `nip.io` address your app will be hosted on.
+  - It will ask for your email, email address, this will be used for any relevant alerts about your HTTPS certificate (you probably won't get any emails from them). Entering your email signifies agreement to the [Let's Encrypt Subscriber Agreement](https://letsencrypt.org/documents/2017.11.15-LE-SA-v1.2.pdf) and the [ZeroSSL Terms of Service](https://zerossl.com/terms/) (either one of those providers may be used to setup your certificate).
```

Comment:
```suggestion
  - It will ask for your email address, which will be used for any relevant alerts about your HTTPS certificate (you probably won't get any emails from them). Entering your email signifies agreement to the [Let's Encrypt Subscriber Agreement](https://letsencrypt.org/documents/2017.11.15-LE-SA-v1.2.pdf) and the [ZeroSSL Terms of Service](https://zerossl.com/terms/) (either one of those providers may be used to setup your certificate).
```

Reply:
Oops, thanks for catching that.